contrib/debian/rules

```diff
@@ -11,10 +11,10 @@ BASE_CONFIG = system/netdata.conf
 
 SYSTEMD_VERSION = $(shell /bin/sh -c "systemd --version 2>&1 | head -n 1 | cut -f 2 -d ' '")
 
-ifeq ($(shell test $(SYSTEMD_VERSION) -ge 235 && echo "1"), 1)
-SYSTEMD_UNIT = $(BUILDDIR)/system/systemd/netdata.service.v235
-else
+ifeq ($(shell test $(SYSTEMD_VERSION) -gt 235 && echo "1"), 1)
```

Comment:
**Probably** using `-le` here is better because if the command fails we will install `$(BUILDDIR)/system/systemd/netdata.service` instead of the old one - this is the logic we use in [install_systemd_service()](https://github.com/netdata/netdata/blob/master/system/install-service.sh.in#L238-L245).

@netdata/agent-sre